demisto_sdk/commands/common/hook_validations/base_validator.py

```diff
@@ -260,6 +261,14 @@ def formatted_error_str():
 
         self.json_output(file_path, error_code, error_message, warning)
         self.add_to_report_error_list(error_code, file_path, FOUND_FILES_AND_ERRORS)
+        if str2bool(os.getenv("GITHUB_ACTIONS")):
+            github_annotation_message = (
+                f"{error_message}\n{suggested_fix}"
+                if suggested_fix
+                else error_message
+            ).replace("\n", "%0A")  # GitHub action syntax
+            print(
+                f"::error file={file_path},line=1,endLine=1,title=Validation Error {error_code}::{github_annotation_message}")
```

Comment:
will this print `error` even if the failure is a `warning`? worth checking, and another unit test 🥲